pyBumpHunter/bumphunter_1dim.py

```diff
@@ -1429,19 +1903,44 @@ def print_bump_info(self):
         Information are printed to stdout.
```

Comment:
Do we need this printing function? In general, I would discourage "print_..." functions, because using `print` is just simple.

I think the golden way is to rather use `__str__` instead of an information object. But more realistically, I would still tend for a `bump_info` or something along the lines: the package should not be a framework that does everything for you but rather a tool for someone who uses Python to give him some functionality.

One problem is that the actual string is now not accessible. So I would anyway refactor this to have primarily a representation (maybe as a string) that can be accessed and _then_ (if really needed) a `print_...` function (but you'll see, that seems rather weird to add it once we have the string info function available).

Do what do you think?

Reply:
This is a good idea, I will think about it.
Maybe just a function that return the string to be printed would be better.

Reply:
Yep, agree